src/components/Gallery/GridImage.css

```diff
@@ -1,10 +1,14 @@
 .ix-gallery-item {
   position: relative;
+  overflow: hidden;
+  border-radius: 4px;
+  background-color: #E7EBEE;
+  border: 1px solid #E7EBEE;
 }
 
 .ix-gallery-item img {
   width: 100%;
-  height: 100%;
```

Comment:
I am leaving 15% of the container space file the filename. This is done for two reasons: 1) to maintain the pre-existing asset size in the grid and 2) to ensure the entire image remains in view rather than "hidden" under the label. This is, AFAIK, consistent with what is done in AM.